api/base/serializers.py

```diff
@@ -274,11 +275,19 @@ class RelationshipField(ser.HyperlinkedIdentityField):
         related_view_kwargs={'node_id': '<_id>', 'wiki_id': '<wiki_pages_current.home>'}
     )
 
+    Field can include optional filters:
```

Comment:
Can you explain further what this does? `develop` currently has a `filter_key` param; how is this different?